app/[locale]/events/components/EventList.tsx

```diff
@@ -0,0 +1,121 @@
+"use client";
+
+import Link from "next/link";
+import { ExternalLink } from "lucide-react";
+import { Button } from "@/components/ui/button";
+import { useState } from "react";
+
+import { Card, CardContent } from "@/components/ui/card";
+import { Separator } from "@/components/ui/separator";
+import eventsData from "@/data/events.json";
+import { cn } from "@/lib/utils";
+
+interface Event {
+  title: string;
+  location: string;
+  date: Date;
+  link: string;
+}
+
+const events: Event[] = eventsData.map((event) => ({
+  ...event,
+  date: new Date(`${event.date}T12:00:00Z`),
+}));
+
+function CalendarIcon({ date, className }: { date: Date; className?: string }) {
+  return (
+    <div
+      className={cn(
+        "relative flex flex-col items-center justify-center w-12 h-12 rounded border bg-background",
+        className
+      )}
+    >
+      <div className="text-[0.65rem] font-medium text-muted-foreground">
+        {date.toLocaleString("default", { month: "short" })}
+      </div>
+      <div className="font-bold text-xl leading-none">{date.getDate()}</div>
+    </div>
+  );
+}
+
+export default function Component() {
```

Comment:
Can we rename `Component` to `EventList` so we don't cause confusion during debugging and stack traces?

```suggestion
export default function EventList() {
```